scipy/optimize/tests/test_hungarian.py

```diff
@@ -1,12 +1,15 @@
-# Author: Brian M. Clapper, G. Varoquaux, Lars Buitinck
-# License: BSD
-
+import pytest
+from scipy.optimize import _hungarian
```

Comment:
Remove this, `_hungarian` is an implementation detail and shouldn't be tested. We want to test the public API

Reply:
actually you can keep it if you need to test `Munkres` which isn't public.